utils/utils.go

```diff
@@ -4,6 +4,8 @@ import (
 	"fmt"
 	"io"
 
+	"github.com/cosmos/cosmos-sdk/types"
```

Comment:
Can you use the `sdktypes` or `sdk` package alias here and then update the comment above the `EvmToCosmosAddress` to reflect the accurate package alias?